app/api_topologies.go

```diff
@@ -50,13 +50,9 @@ var (
 )
 
 // namespaceFilters generates a namespace selector option group based on the given namespaces
-func namespaceFilters(namespaces []string, defaultNamespace, noneLabel string) APITopologyOptionGroup {
-	options := APITopologyOptionGroup{ID: "namespace", Default: "", SelectType: "union", NoneLabel: noneLabel}
+func namespaceFilters(namespaces []string, noneLabel string) APITopologyOptionGroup {
+	options := APITopologyOptionGroup{ID: "namespace", Default: "none", SelectType: "union", NoneLabel: noneLabel}
```

Comment:
The `"none"` here took me a while to figure out; it is a dependency on a constant in the UI code.

I initially had `""` and then `noneLabel`, neither of which worked.